### ISSUE TITLE
current git fails to compile 

### ISSUE THREAD

**Issue by stefson** [Return-To-The-Roots/s25client#1396]
most likely related to https://github.com/Return-To-The-Roots/s25client/pull/1385

```
make[2]: *** [libs/s25main/CMakeFiles/s25Main.dir/build.make:1954: libs/s25main/CMakeFiles/s25Main.dir/figures/nofArmorer.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBaker.cpp: In member function ‘virtual void nofBaker::DrawWorking(DrawPoint)’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBaker.cpp:92:13: error: ‘gwg’ was not declared in this scope
   92 |             gwg->GetSoundMgr().playNOSound(68, *this, now_id);
      |             ^~~
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBaker.cpp:108:13: error: ‘gwg’ was not declared in this scope
  108 |             gwg->GetSoundMgr().playNOSound(68, *this, now_id);
      |             ^~~
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBrewer.cpp: In member function ‘virtual void nofBrewer::DrawWorking(DrawPoint)’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBrewer.cpp:47:9: error: ‘gwg’ was not declared in this scope
   47 |         gwg->GetSoundMgr().playNOSound(51, *this, 0);
      |         ^~~
make[2]: *** [libs/s25main/CMakeFiles/s25Main.dir/build.make:1980: libs/s25main/CMakeFiles/s25Main.dir/figures/nofBaker.cpp.o] Error 1
make[2]: *** [libs/s25main/CMakeFiles/s25Main.dir/build.make:1993: libs/s25main/CMakeFiles/s25Main.dir/figures/nofBrewer.cpp.o] Error 1
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBuilder.cpp: In member function ‘virtual void nofBuilder::HandleDerivedEvent(unsigned int)’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBuilder.cpp:208:13: error: ‘gwg’ was not declared in this scope
  208 |             gwg->GetSoundMgr().stopSounds(*this);
      |             ^~~
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBuilder.cpp: In member function ‘virtual void nofBuilder::Draw(DrawPoint)’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBuilder.cpp:348:17: error: ‘gwg’ was not declared in this scope
  348 |                 gwg->GetSoundMgr().playNOSound(soundId, *this, index, 160 - rand() % 60);
      |                 ^~~
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBuildingWorker.cpp: In member function ‘void nofBuildingWorker::LostWork()’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBuildingWorker.cpp:276:13: error: ‘gwg’ was not declared in this scope
  276 |             gwg->GetSoundMgr().stopSounds(*this);
      |             ^~~
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofBuildingWorker.cpp:298:13: error: ‘gwg’ was not declared in this scope
  298 |             gwg->GetSoundMgr().stopSounds(*this);
      |             ^~~
make[2]: *** [libs/s25main/CMakeFiles/s25Main.dir/build.make:2006: libs/s25main/CMakeFiles/s25Main.dir/figures/nofBuilder.cpp.o] Error 1
make[2]: *** [libs/s25main/CMakeFiles/s25Main.dir/build.make:2019: libs/s25main/CMakeFiles/s25Main.dir/figures/nofBuildingWorker.cpp.o] Error 1
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofCarpenter.cpp: In member function ‘virtual void nofCarpenter::DrawWorking(DrawPoint)’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofCarpenter.cpp:45:9: error: ‘gwg’ was not declared in this scope
   45 |         gwg->GetSoundMgr().playNOSound(54 + ((now_id) % 8) / 4, *this, now_id / 4);
      |         ^~~
make[2]: *** [libs/s25main/CMakeFiles/s25Main.dir/build.make:2045: libs/s25main/CMakeFiles/s25Main.dir/figures/nofCarpenter.cpp.o] Error 1
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofButcher.cpp: In member function ‘virtual void nofButcher::DrawWorking(DrawPoint)’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofButcher.cpp:44:9: error: ‘gwg’ was not declared in this scope
   44 |         gwg->GetSoundMgr().playNOSound(66, *this, now_id / 6);
      |         ^~~
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofCarrier.cpp: In member function ‘virtual void nofCarrier::Draw(DrawPoint)’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofCarrier.cpp:315:21: error: ‘gwg’ was not declared in this scope
  315 |                     gwg->GetSoundMgr().playNOSound(84, *this, 0);
      |                     ^~~
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofCarrier.cpp: In member function ‘virtual void nofCarrier::Walked()’:
/var/tmp/portage/games-strategy/s25rttr-9999-r5/work/s25rttr-9999/libs/s25main/figures/nofCarrier.cpp:334:9: error: ‘gwg’ was not declared in this scope
  334 |         gwg->GetSoundMgr().stopSounds(*this);
      |         ^~~
make[2]: *** [libs/s25main/CMakeFiles/s25Main.dir/build.make:2032: libs/s25main/CMakeFiles/s25Main.dir/figures/nofButcher.cpp.o] Error 1


```

[build.log.zip](https://github.com/Return-To-The-Roots/s25client/files/6165959/build.log.zip)

**Comment by stefson** [comment]
this is on amd64 linux with gcc-9.3.0

**Comment by Flamefire** [comment]
Fixed in #1391
